src/main/scala/com/example/ParkingS/controllers/DriverController.scala

```diff
@@ -0,0 +1,116 @@
+package com.example.ParkingS.controllers
+
+import java.time.LocalDateTime
+
+import com.example.ParkingS.exceptions.{IncorrectBeginParkingRequestException, _}
+import com.example.ParkingS.model.Ticket
+import com.example.ParkingS.repository.{CurrencyFakeRepository, TicketFakeRepository}
+import com.example.ParkingS.requests.{BeginParkingRequest, EndParkingRequest}
+import com.example.ParkingS.responses.{BeginParkingResponse, EndParkingResponse, ErrorResponse}
+import com.example.ParkingS.service.price.PriceCalculator
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation._
+
+@RestController
+@RequestMapping(Array("/driver"))
+class DriverController @Autowired()(private val ticketRepository: TicketFakeRepository,
+                                    private val currencyRepository: CurrencyFakeRepository,
+                                    private val priceCalculator: PriceCalculator) {
+    
+    @PostMapping(Array("/start"))
+    def beginParking(@RequestBody beginParkingRequest: BeginParkingRequest): BeginParkingResponse = {
```

Comment:
w kwestii organizacji metody to jest taka bardziej java niż scala... tylko tyle, że składnia jest scalowa